src/QMCDrivers/DMC/WalkerControlMPI.cpp

```diff
@@ -186,9 +161,8 @@ void WalkerControlMPI::swapWalkersSimple(MCWalkerConfiguration& W)
   std::vector<int> minus, plus;
```

Comment:
Description of algorithm needed in doxygen comment

Reply:
The doxygen comment for this routine will be added in the next PR since the algorithm will further change.